tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,77 @@
+"""Task 1 and 2 of Module 2."""
+
+stuff = {'rope': 1, 'torch': 6, 'gold coin': 42, 'dagger': 1, 'arrow': 12}
+
+dragonLoot = [
+
+    'gold coin', 'chewed gum', 'dagger', 'gold coin', 'gold coin', 'ruby',
+
+    'rubbish', 'chewed gum', 'used tissue']
+
+
+def display_inventory(inventory):
+    """Display inventory.
```

Comment:
I would rather skip repeating the function name in docstring as it does not add more info. Maybe just merge it with the second sentence?